src/sock.c

```diff
@@ -310,6 +314,88 @@ static int listen_on_one_socket(struct addrinfo *ad)
         return listenfd;
 }
 
+#ifdef SO_BINDTODEVICE
+static int listen_on_interface(const char *interface, uint16_t port)
+{
+        int listenfd;
+        int ret;
+        int ipv6=1;
```

Comment:
```suggestion
        int ipv6 = 1;
```

Reply:
@BlankEclair: please refrain on suggesting whitespace changes on random PRs you absolutely are not involved with. if i want such changes, i will ask for them myself.

Reply:
Sorry--I overzealously commented on the inconsistency

It won't happen again